<!-- -*- mode: XML -*- -->
<launch>
  <arg name="hand_config" default="$(find wsg_50_driver)/config/hand0.yaml" />
  <rosparam file="$(arg hand_config)" command="load" />

  <node pkg="wsg_50_driver" type="test_hand_read" name="test_hand_read" output="screen"/>

</launch>

